db/schema.rb

```diff
@@ -369,10 +369,12 @@
     t.uuid "default_site_banner_id"
     t.boolean "default_site_banner_mode", default: false, null: false
     t.uuid "uphold_id"
+    t.uuid "uphold_connection_id"
```

Comment:
Unused code

Reply:
I added this because in the `uphold_connection` model we call `publisher` which makes a call to the DB to get this information.

Reply:
Sorry I didn't realize that you added a reference. That being said, my impression is that having rows that point at each other could result in bad state. Preferably we just establish a `belongs_to` and `has_one` in the models rather than store this in the database.

Reply:
Unified it here ✔️ 
https://github.com/brave-intl/publishers/pull/1825/commits/6df08b7dd73c63e4ce15c81926fa5f693979efaa